docs/source/release_notes/release-notes.rst

```diff
@@ -44,6 +44,10 @@ develop
          - Cassandra now attempts to truncate when performing a ``deleteRange(RangeRequest.All())`` in an effort to build up less garbage.
            (`Pull Request <https://github.com/palantir/atlasdb/pull/1617>`__)
 
+    *    - |fixed|
```

Comment:
nit: `|improved|`? Although the previous heap settings probably wouldn't have been good for perf (since we'd need to resize the heap a few times) I wouldn't call it broken.